docs/components/connectors/use-connectors/index.md

```diff
@@ -158,7 +158,7 @@ bpmnError("123", "error received")
 
 #### HTTP errors to BPMN errors
 
-Using the [REST Connector](/components/connectors/protocol/rest.md), you can handle HTTP errors directly in your business process model:
+Using the [REST Connector](/components/connectors/protocol/rest.md), you can handle HTTP errors directly in your business process model by setting a Header named `errorExpression` with the following value:
```

Comment:
Instead of two separate sentences ending in a colon, I consolidated them into one, single sentence still ending in a colon

Reply:
Thanks